test/_test_log_capture.jl

```diff
@@ -5,7 +5,7 @@ const log_display = Symbol(ENV["LOG_DISPLAY"])
 @testset "log capture logs=$(repr(log_display))" begin
     @testset "TestItem" begin
         @testset "log capture for println" begin
-            ti = @testitem "uses println" begin
+            ti = @testitem "uses println" _quote=true begin
```

Comment:
this `_quote` keyword is how we now avoid running a `@testitem` immediately when outside of `runtests`. i don't think _users_ should ever need this (only developers of the package)